http.go

```diff
@@ -100,6 +105,9 @@ type Client struct {
 
 	// NewNotifyClient initializes these services
 	Credentials *NotifyCredentialsService
+
+	// NewLookupClient initializes these services
+	Lookups *LookupsService
```

Comment:
This is a little confusing but I might make this one PhoneNumbers (assuming that doesn't conflict with other resource names)... then to access the Lookups API you'd do

```go
client.Lookups.PhoneNumbers.Get("+1...")
```

which matches the URL and the other API's a little more closely.